source/optional_features/custom_error_pages/index.html.md.erb

```diff
@@ -0,0 +1,24 @@
+---
+title: Custom error pages 
+weight: 135
+---
+
+# Custom error pages
+
+GOV.UK Pay supports custom error pages.
+
+When a payment reaches a terminal, unsuccessful state, GOV.UK Pay will
+immediately redirect the user to the return URL (`return_url`), which you can
+set in the response to the `POST /v1/payments` API call. 
+
+To set this up for your service, you should make a query to the GOV.UK Pay API
+to determine what the failure state is, and set an appropriate error message. 
+
+If you do not use custom error pages, users will see generic unsuccessful
+payment screens.
+
+## Expired payments 
+
+If the user abandons a payment and it expires, GOV.UK Pay will only redirect
+the user if they did not actually leave the payment screen. If they try to
+continue it, they will be returned to the service. 
```

Comment:
might want to mention here that this is why we still expect the service to have a clean-up job if they use this feature

Reply:
Can we reasonably expect the service (reader of the documentation) to know what will be meant by the term "clean-up job", and how to use it? Are any other terms likely to be used?

Reply:
mop up job ;)